examples/specs/wheat_wages.vl.json

```diff
@@ -0,0 +1,167 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v3.json",
+  "width": 800,
+  "height": 400,
+  "data": { "url": "data/wheat.json"},
+  "transform": [{"calculate": "+datum.year + 5", "as": "year_end"}],
+  "layer": [
+    {
+      "mark": {
+        "type": "bar"
+      },
+      "encoding": {
+        "x": {
+          "field": "year",
+          "type": "quantitative",
+          "axis": {
+            "tickCount": 5,
+            "format": "d"
+          }
+        },
+        "x2": {
+          "field": "year_end"
+        },
+        "y": {
+          "field": "wheat",
+          "type": "quantitative",
+          "axis": { "zindex": 1 }
+        },
+        "fill": { "value": "#AAAAAA" },
+        "stroke": { "value": "#999999" }
+      }
+    },
+    {
+      "data": {
+        "values": [
+          { "year": "1600" },
+          { "year": "1650" },
+          { "year": "1700" },
+          { "year": "1750" },
+          { "year": "1800" }
+        ]
+      },
+      "mark": "rule",
+      "encoding": {
+        "x": {
+          "field": "year",
+          "type": "quantitative"
+        },
+        "stroke": { "value": "#000" },
+        "strokeWidth": { "value": 0.6 },
+        "opacity": { "value": 0.5 }
+      }
+    },
+    {
+      "mark": "area",
+      "encoding": {
+        "x": {
+          "field": "year",
+          "type": "quantitative"
+        },
+        "y": {
+          "field": "wages",
+          "type": "quantitative"
+        },
+        "color": { "value": "#a4cedb" },
+        "opacity": { "value": 0.7 }
+      }
+    },
+    {
+      "mark": "line",
+      "encoding": {
+        "x": {
+          "field": "year",
+          "type": "quantitative"
+        },
+        "y": {
+          "field": "wages",
+          "type": "quantitative"
+        },
+        "color": { "value": "#000" },
+        "opacity": { "value": 0.7 }
+      }
+    },
+    {
+      "mark": {
+        "type": "line",
+        "yOffset": -2
+      },
+      "encoding": {
+        "x": {
+          "field": "year",
+          "type": "quantitative"
+        },
+        "y": {
+          "field": "wages",
+          "type": "quantitative"
+        },
+        "color": { "value": "#EE8182" }
```

Comment:
Make the spec more concise by replacing value defs with mark properties?

Reply:
Done